doc/platforms.md

```diff
@@ -12,15 +12,9 @@ Flame keeps it support on the stable and beta channel, dev and master channel sh
 
 To use Flame on web you need to be on the beta channel of Flutter and the game should be running using skia rendering, otherwise weird side effects may happen.
 
-To run your game using skia in release mode, use the following command:
+To run your game using skia, use the following command:
```

Comment:
maybe link to the skia repo so people can read more? not obviously clear what this is for new people